apps/base-docs/tutorials/docs/5_shopify-storefront-commerce.md

```diff
@@ -0,0 +1,119 @@
+
+#### Oxygen
+
+Oxygen is Shopify’s global serverless hosting platform for deploying Hydrogen storefronts at the edge, managing deployment environments, environment variables, caching, and integrating with Shopify’s CDN.
+
+## Getting Started
+
+To comply with Shopify's requirements, the Coinbase Commerce account linked to Shopify must be a Coinbase Managed account.
+
+To create one, [sign up](https://beta.commerce.coinbase.com/sign_up) for a business account and select "Coinbase Commerce" as the account type. See this Coinbase Commerce Merchant [help article](https://help.coinbase.com/en/commerce/getting-started/merchant-guidelines-commerce-application) for additional information on how to sign your business up.
+
+### Link Coinbase Commerce to your Shopify store
+
+![shopify-install-commerce.gif](../../assets/images/shopify-storefront-commerce/shopify-install-commerce.gif)
+
+Once, you've created a Coinbase Commerce Managed account you can now add the plugin to your Shopify store.
+
+To add Coinbase Commerce as a payment method on your Shopify store, start by navigating to your admin page at `https://admin.shopify.com/store/<YOUR-STORE-NAME>`.
+Once there, click on Settings , located in the bottom left panel of the screen (or at `https://admin.shopify.com/store/<YOUR-STORE-NAME>settings/general`). Then select the “Payments” tab.
+Proceed by clicking “Add a payment method” and choose “Search by provider.” In the search field, type in “Coinbase” and select “Coinbase Commerce” from the results.
+Click "Install" and you should be redirected to Coinbase Commerce with aprompt log into your Coinbase account. Finally, click "Activate" to enable the Coinbase Commerce plugin.
+
+### Create a Storefront
+
+In this tutorial, we'll guide you through the steps to create a new Hydrogen storefront for your Shopify store. This will allow you to showcase the products you already have in your Shopify account. Make sure you've already created your products by following the [Shopify products guide](https://help.shopify.com/en/manual/online-sales-channels/shop/products-and-collections).
+
+To get started, we'll first clone a Shopify demo store using the Hydrogen framework. This will give us a quick setup to work with.
+
+```bash
+npm create @shopify/hydrogen@latest -- --quickstart
+```
+
+Once the demo store is cloned, navigate into the project directory and start the development server:
+
+```bash
+cd hydrogen-quickstart
+npx shopify hydrogen dev
+```
+
+Next, you'll need to link your Hydrogen project to your Shopify store. This step connects the demo storefront with your Shopify account, allowing you to display your products.
+
+```bash
+npx shopify hydrogen link
+```
+
+To ensure your storefront is properly configured, update the project environment variables. This pulls the necessary settings from your Shopify account into the Hydrogen project.
```

Comment:
I'd say that it's going to open the browser and you should log in to your account.

Reply:
And explain what you need to do.  I almost missed the "? Select a shop to log in to:" step.

Reply:
Then I got:

```
│                                                                                              │
│  The Admin GraphQL API responded unsuccessfully with errors:                                 │
│                                                                                              │
│  [                                                                                           │
│    {                                                                                         │
│      "message": "Access denied for hydrogenStorefronts field. Required access: Request must  │
│   be initiated from the Shopify CLI and user must have full access to apps or access to the  │
│   Hydrogen channel.",                                                                        │
│      "locations": [                                                                          │
│        {                                                                                     │
│          "line": 3,                                                                          │
│          "column": 5                                                                         │
│        }                                                                                     │
│      ],                                                                                      │
│      "path": [                                                                               │
│        "hydrogenStorefronts"                                                                 │
│      ],                                                                                      │
│      "extensions": {                                                                         │
│        "code": "ACCESS_DENIED",                                                              │
│        "documentation": "https://shopify.dev/api/usage/access-scopes",                       │
│        "requiredAccess": "Request must be initiated from the Shopify CLI and user must have  │
│   full access to apps or access to the Hydrogen channel."                                    │
│      }                                                                                       │
│    }                                                                                         │
│  ]                                                                                           │
│                                                                                              │
│  Request ID: 0d1fa62c-94c3-486f-8f04-2c5967f501f2-1719239581      
```

Reply:
It's unclear what you mean by " update the project environment variables"

Please state exactly what to do here!

Reply:
Oh, I see now.  I use colons so the reader connects the sentence to the following code block.